mem0/vector_stores/azure_ai_search.py

```diff
@@ -210,7 +250,8 @@ def get(self, vector_id) -> OutputData:
         return OutputData(id=result["id"], score=None, payload=json.loads(result["payload"]))
 
     def list_cols(self) -> List[str]:
-        """List all collections (indexes).
+        """
+        List all collections (indexes).
 
         Returns:
             List[str]: List of index names.
```

Comment:
You could use `index_client.list_index_names()` that pulls just the names of the indexes and not the entire schema for each.